### PR TITLE
add example showing how to customize figure subplot titles

### PR DESCRIPTION
**Pull request by jdamiba**
The purpose of this PR is to add an example which demonstrates how to customize figure subplot titles. 

see https://github.com/plotly/plotly_express/issues/36

### CODE REVIEW

File: doc/python/facet-plots.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ jupyter:
     page_type: u-guide
     permalink: python/facet-plots/
     redirect_from:
-      - python/trellis-plots/
-      - python/facet-trellis/
+    - python/trellis-plots/
+    - python/facet-trellis/
```
Comment (emmanuelle) [inline doc/python/facet-plots.md]: did you change this yourself or was it Jupyter? From a quick `grep` I think we have the two syntaxes in the doc directory, so this should be fine, but just checking.

**Comment by emmanuelle** [comment]
One little thing I was not sure of in the header, but otherwise :dancer:

File: doc/python/facet-plots.md (hunk 1 above)
Comment (jdamiba) [reply doc/python/facet-plots.md]: It was Jupyter, not me that edited the front matter.

Comment (emmanuelle) [reply doc/python/facet-plots.md]: ok, :dancer:!

**Pull request merged**